### DISCUSSION TITLE
ChatGPT 微信技术交流群

### DISCUSSION THREAD

**Discussion by lencx** [lencx/ChatGPT#276] [Announcements]
**注意：根据需要扫微信二维码，不要重复多次进群**

插播一条信息：我失业了，目前正在找工作，坐标上海，职位前端开发，如果有合适机会可以私信我，GitHub 首页有联系方式。

👇 ChatGPT 常见问题交流可以扫此码（二维码过期可以关注公众号“浮之静”，发送"chat" 进群）：

<img width="240" src="https://user-images.githubusercontent.com/16164244/218293528-b1088837-833d-4227-99d2-7d2887fc60a8.JPG" />

<img width="300" src="https://user-images.githubusercontent.com/16164244/218381206-5d14c065-1033-4de2-8c7a-826d574a7fec.png" />

**Comment by wayner6** [comment]
微信扫描不出来是怎么回事噻？总是提示“识物”

**Reply by lencx** [reply]
可以扫呀，你是不是姿势不对

**Reply by wayner6** [reply]
ok了，可能是手机神经了

**Reply by Java2Boy** [reply]
1：超过200，需邀请
2：已过期

**Reply by cnscorpions** [reply]
同上，能拉一下群吗?

**Reply by lencx** [reply]
去首页关注公众号，回复：“chat”

**Reply by maojianyou** [reply]
这个二维码过期了，想咨询一下这个桌面版本跟网页版本有啥优势，有了网页版本，为什么还要用桌面版本呢

**Comment by dimikarl2022** [comment]
sygma

**Reply by szwuzy** [reply]
哈哈，是个收费码

**Comment by tong0401** [comment]
有可用的chatGPT号，要的来

**Reply by taoyuanhang** [reply]
有plus吗？有的话告诉我。有个人账号吗？不是共享的那种。
wx：tyhmoshu1
qq：1608876448

**Comment by zhuanshiyan** [comment]
二維碼失效了，能不能重新發個

**Reply by lencx** [reply]
这里不更新了，太麻烦，关注公众号，发“chat”就可以看到进群二维码。

**Reply by justinscript** [reply]
公众号名字是什么呢?

**Reply by lencx** [reply]
浮之静

**Comment by wdf186** [comment]
你微信被加爆了

**Comment by esncy** [comment]
期待能出中文版。

**Comment by qingwa007** [comment]
这个卡好用，也有群可以交流  https://chatgpt007.wordpress.com/2023/07/13/chatgpt-plus-2/
